src/output.c

```diff
@@ -769,3 +915,176 @@ void OutputNotifyFileRotation(void) {
         *(flag->flag) = 1;
     }
 }
+
+TmEcode OutputLoggerLog(ThreadVars *tv, Packet *p, void *thread_data,
+    PacketQueue *pq, PacketQueue *postpq)
```

Comment:
I think PacketQueue *pq, PacketQueue *postpq can both be removed. No output should be creating pseudo packets.